src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline

```diff
@@ -930,7 +976,7 @@ public class LibraryServiceSettings extends ServiceApiSettings {
     /**
      * Returns the builder for the settings used for calls to listShelves.
      */
-    public PageStreamingCallSettings.Builder<ListShelvesRequest, ListShelvesResponse, Shelf> listShelvesSettings() {
+    public PageStreamingCallSettings.Builder<ListShelvesRequest, ListShelvesResponse, Shelf, ListShelvesPagedResponse> listShelvesSettings() {
```

Comment:
This looks out of date. Maybe generate into api-client-staging and run the java tests to make sure everything is consistent.